ocp4/profiles/ncp.profile

```diff
@@ -1,19 +1,24 @@
 documentation_complete: true
```

Comment:
This is meant to be FedRAMP moderate. but API/platform specific checks. let's not change this specific profile

Reply:
nevermind, GitHub just showed the diff weirdly